scripts/convert_phonemizer_onnx.py

```
@@ -58,19 +58,15 @@ def generate(self, batch: dict[str, torch.Tensor]) -> tuple[torch.Tensor, torch.
 
 
 class ForwardTransformer(Model):
-    def __init__(
-        self, encoder_vocab_size: int, decoder_vocab_size: int, d_model=512, d_fft=1024, layers=4, dropout=0.1, heads=1
-    ) -> None:
+    def __init__(self, encoder_vocab_size: int, decoder_vocab_size: int, d_model=512, d_fft=1024, layers=4, dropout=0.1, heads=1) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `d_model`

(ANN001)